docs/drivers.rst

```diff
@@ -52,6 +52,7 @@ These libraries build on top of the low level APIs to simplify common tasks.
     Framebuf Module <https://circuitpython.readthedocs.io/projects/framebuf/en/latest/>
     Image Load <https://circuitpython.readthedocs.io/projects/imageload/en/latest/>
     InfraRed Remote <https://circuitpython.readthedocs.io/projects/irremote/en/latest/>
+    IterTools <https://circuitpython.readthedocs.io/projects/itertools/en/latest/>
```

Comment:
@dastels The link in `drivers.rst` to the IterTools RTD does not find an active project page: https://circuitpython.readthedocs.io/projects/itertools/en/latest/

Did you set up RTD for this library?

Reply:
Oh... that's not done by the build process?  I assumed it was generated from all the doc directory stuff. I just copy pasted.

Reply:
No, it is not done by the build process, it requires manual setup on RTD. @brentru Please assist @dastels with getting it to the furthest point you are able. Then file an issue on the IterTools repo and tag me so I can finish it up. Thanks!

For now, this PR can be merged.